### DISCUSSION TITLE
posts/2024/04/life-intrudes/

### DISCUSSION THREAD

**Discussion by giscus** [brycewray/comments#48] [Announcements]
# posts/2024/04/life-intrudes/

My normal existence screeches to an unexpected halt after some equally unexpected news.

https://www.brycewray.com/posts/2024/04/life-intrudes/

**Comment by rodrigoalcarazdelaosa** [comment]
Really sorry to hear this, Bryce. Hope you'll get better soon. My best wishes, Rodri.

**Reply by brycewray** [reply]
@rodrigoalcarazdelaosa Thank you, sir. I share that hope. 🙂
